src/types/Filter.ts

```diff
@@ -0,0 +1,7 @@
+export enum FILTER {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
+
+export type FilterStatus = FILTER.ALL | FILTER.ACTIVE | FILTER.COMPLETED;
```

Comment:
It's the wrong usage of an enum. You don't need `FilterStatus` union. The enum is great because you can use it as type and also as values. So instead of `FilterStatus` use `Filter` enum everywhere